xml/obs_source_service.xml

```diff
@@ -284,15 +279,19 @@
     </listitem>
    </varlistentry>
    <varlistentry>
+    <term><literal>manual</literal> Mode</term>
     <term><literal>disabled</literal> Mode</term>
     <listitem>
      <para>
-      The disabled mode is neither running the service locally nor on the
-      server side. It can be used to temporarily disable the service but keeping
+      The manual mode is neither running the service locally nor on the
+      server side by default. It can be used to temporarily disable the service but keeping
       the definition as part of the service definition. Or it can be used to
       define the way how to generate the sources and doing so by manually calling
-      <command>osc service runall</command> The result will
-      get committed as standard files again.
+      <command>osc service rundisabled</command>.
+      The result will get committed as standard files again.
+      NOTE: it did only exist as "disabled" before OBS 2.11, but "manual" is the better
+            matching alias name for its usage. The osc client may do have different
+            behaviour in future between manual and disabled.
      </para>
```

Comment:
This would probably be better as a real note:

```suggestion
     </para>
     <note>
      <title>This Valie Is an Alias for "disabled"</title>
      <para> 
       This option was called "disabled" in earlier versions of OBS.
       In OBS 2.11, it was aliased to "manual" to better reflect usage.
       In the future, the osc client may have different behavior in future
       between "manual" and "disabled".
     </para>
    </note>
```